README.md

```diff
@@ -67,6 +67,15 @@ The full spec suite can be run in docker locally. There are several ways to do t
 docker-compose exec web rake
 ```
 
+#### Docker with an M1 chip
+- You may run into issues with Selenium with an M1 chip. If so, use the override file provided called `docker-compose-override.yml`
```

Comment:
these will potentially need to be updated, depending on if we want an override file or just to change it normally